src/server/api/reply.ts

```diff
@@ -0,0 +1,68 @@
+import { Type } from '@sinclair/typebox'
+import type { APIConfig, FastifyTypebox } from '.'
+import type Store from '../store'
+import type ActivityPubSystem from '../apsystem.js'
+
+export const replyRoutes = (cfg: APIConfig, store: Store, apsystem: ActivityPubSystem) => async (server: FastifyTypebox): Promise<void> => {
+  // Get list of replies for a post
+  server.get<{
+    Params: {
+      actor: string
+      postURL: string
+    }
+    Reply: string | string[]
+  }>('/:actor/replies/:postURL', {
+    schema: {
+      params: Type.Object({
+        actor: Type.String(),
+        postURL: Type.String() // This will represent the post's URL to fetch the replies for
+      }),
+      response: {
+        200: Type.Array(Type.String()),
+        409: Type.String()
+      },
+      description: 'Get list of replies for a post.',
+      tags: ['Replies']
+    }
+  }, async (request, reply) => {
+    const { actor, postURL } = request.params
+    const allowed = await apsystem.hasPermissionActorRequest(actor, request)
```

Comment:
@fauno @gobengo what do you think about us leaving this api unauthenticated so that we can optionally have the published activitypub Notes point to the replies URL to the inbox instead of pulling the json and republishing each time?

E.g. 

```json
{
    "@context": "https://www.w3.org/ns/activitystreams",
    "id": "https://staticpub.mauve.moe/helloworld.jsonld",
    "type": "Note",
    "published": "2023-06-29T17:29:45Z",
    "url": "https://staticpub.mauve.moe/helloworld",
    "attributedTo": "https://staticpub.mauve.moe/about.jsonld",
    "to": [
        "https://www.w3.org/ns/activitystreams#Public"
    ],
    "content": "Hello World!",
    "replies":  "https://social.distributed.press/v1/@mauve@staticpub.mauve.moe/replies/https%3A%2F%2Fstaticpub.mauve.moehelloworld-replies.jsonld/"
}
```